cpp/src/link_analysis/pagerank.cu

```diff
@@ -382,3 +382,39 @@ template void pagerank<int, int, double>(experimental::GraphCSCView<int, int, do
                                          bool has_guess);
 
 }  // namespace cugraph
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+namespace cugraph {
+
+template <typename VT, typename ET, typename WT>
+void mg_pagerank_temp(raft::handle_t &handle,
```

Comment:
Let's make this match the C++ API in PR #947 so that it is compatible from the start.

Reply:
updated